### ISSUE TITLE
Dataset properties

### ISSUE THREAD

**Issue by dionstew** [JemuelStanley47/PoseCNN#2]
Hi, 
Are you still working on this repo? 

I'd like to use my own collected dataset and fine-tune the pre-trained the poseCNN model created in .ipynb file in this repo. 
I understand that I should collect the RGB image, segmentation mask, and depth image. I can not figure out the what are the ground truth properties written in "train_gt.json" and "train_gt_info.json" files.  

Could you please explain the dataset properties mentioned train_gt or/and train_gt_info please?
